sbt_common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Work.scala

```diff
@@ -42,6 +42,7 @@ case class UnidentifiedWork(
   sourceIdentifier: SourceIdentifier,
   version: Int,
   title: String,
+  identifiedType: String = classOf[IdentifiedWork].getSimpleName,
```

Comment:
Does this need to be passed around as a case class parameter? Could we put it as a static value on the body of the class instead?

Reply:
Ah, because we need it on the JSON for the ID minter? Might be worth a comment.